site/cds_rdm/inspire_harvester/jobs.py

```diff
@@ -99,6 +104,11 @@ def validate_exclusive_arguments(self, data, **kwargs):
             )
 
 
+def callback(result):
+    """Callback function."""
+    print(f"Result: {result}")
```

Comment:
let's remove for now, since we want to merge this PR
```suggestion
def callback(result):
    """Callback function."""
    print(f"Result: {result}")
```